vars/README.md

```diff
@@ -38,3 +38,6 @@ blueprints include:
 * `main-ueransim.yml`: Configures UERANSIM in place of gNBsim,
   providing a second way to direct workload at the Core. Details documented
   [here](https://docs.aetherproject.org/master/onramp/blueprints.html#ueransim).
+
+* `main-sriov.yml`: Configures UPF in DPDK/SRIOV mode. It will help to improve
+  UPF packet processing performance
```

Comment:
Are you planning to make changes/updates to the Aether documentation (similarly as it is done with the other blueprints)? If so, I think you need to open a PR there as well and have the link to it added here

Reply:
Yes. I plan to thoroughly document these variables in the blueprints section. My main "ask" is that we enumerate the assumptions... for example about how the NIC is set up. Such comments belong in the values_file -- which has all the hardware-specific parameters -- and probably not in the vars file (except to point to the relevant values-file.